TilerServer/TilerWebModule.pas

```diff
@@ -3646,39 +3674,16 @@ function TSliceDiffGeometryICLR.GenerateTileCalc(const aExtent: TExtent; aBitmap
             else
             begin
               //see if we can find L width
-              if not IsNaN(isgop.Value.value) then
-              begin
-                if not IsNaN(refObj.value)
-                then widthL := (isgop.Value.value + refObj.value) / 2
-                else widthL := isgop.Value.value;
-              end
-              else
-              begin
-                if not IsNaN(refObj.value)
-                then widthL := refObj.value
-                else validL := False;
-              end;
+              widthL := CalculateWidth(isgop.Value.value, refObj.value, validL);

```

Comment:
'isgop' is does not seem like a good variable name for  CurrentScenario.

Reply:
Fixed!